tests/test_parse_command.py

```
@@ -10,78 +9,40 @@
 
 @pytest.fixture
 def parse_test_env():
-    """Set up test environment for parse command tests."""
-    # Create a temporary directory
-    temp_dir = tempfile.mkdtemp()
+    """Create a temporary directory for testing parse command."""
+    with tempfile.TemporaryDirectory() as tmp_dir:
+        # Create a sample JSON log file
+        log_file = os.path.join(tmp_dir, "test.log")
+        with open(log_file, "w") as f:
+            f.write('{"timestamp": "2023-01-01T00:00:00Z", "level": "info", "message": "Test log", "code": 200}\n')
```

Pylint (3.3.7)

[convention] Line too long (115/100)

(C0301)